packages/server/graphql/mutations/helpers/isStartMeetingLocked.ts

```diff
@@ -1,12 +1,30 @@
+import isValid from '../../../graphql/isValid'
+import getTeamsByOrgIds from '../../../postgres/queries/getTeamsByOrgIds'
 import {DataLoaderWorker} from '../../graphql'
 
 const isStartMeetingLocked = async (teamId: string, dataLoader: DataLoaderWorker) => {
   const team = await dataLoader.get('teams').loadNonNull(teamId)
-  const {isPaid, lockMessageHTML} = team
-  if (isPaid) return null
-  return lockMessageHTML
-    ? 'Wow, you’re determined to use Parabol! That’s awesome! Do you want to keep sneaking over the gate, or walk through the door with our Sales team?'
-    : 'Sorry! We are unable to start your meeting because your team has an overdue payment'
+  const {lockMessageHTML, orgId, tier} = team
+  const orgTeams = await getTeamsByOrgIds([orgId])
+  const anyOrgTeamIsLocked = orgTeams.some((team) => !team.isPaid)
+  if (tier !== 'personal' && anyOrgTeamIsLocked) {
+    // if this team was manually locked, be mean because they called this by hiding the modal
+    return lockMessageHTML
+      ? 'Wow, you’re determined to use Parabol! That’s awesome! Do you want to keep sneaking over the gate, or walk through the door with our Sales team?'
+      : 'Sorry! We are unable to start your meeting because your organization has an overdue payment'
+  }
+  // if this team wasn't manually locked, see if any of its members are on locked teams
+  const teamMembers = await dataLoader.get('teamMembersByTeamId').load(teamId)
+  const userIds = teamMembers.map(({userId}) => userId)
+  const allTeamMembers = (await dataLoader.get('teamMembersByUserId').loadMany(userIds))
+    .filter(isValid)
+    .flat()
+  const teamIds = [...new Set(allTeamMembers.map(({teamId}) => teamId))]
+  const allRelatedTeams = (await dataLoader.get('teams').loadMany(teamIds)).filter(isValid)
+  const anyTeamIsLocked = allRelatedTeams.some((team) => team.isPaid === false)
```

Comment:
Rather than checking `team.lockMessageHTML`, I think we should check `team.isPaid` because we can lock a team without including the `lockMessageHTML`.

If a team has paid but a team member belongs to a locked team, the paying team won't be able to start a meeting. I think that's OK - we could remove the team member or unlock the locked team if we think the paying team should be able to start a meeting.

Reply:
makes sense!